packages/libs/react-ui/src/components/Button/index.tsx

```diff
@@ -1,15 +1,4 @@
-import { Button as ButtonContainer, IButtonProps } from './Button';
-import { ButtonIcon, IButtonIconProps } from './ButtonIcon';
-
-import { FC } from 'react';
+import { Button, IButtonProps } from './Button';
 
 export { IButtonProps };
-interface IButton {
-  Root: FC<IButtonProps>;
-  Icon: FC<IButtonIconProps>;
-}
-
-export const Button: IButton = {
-  Root: ButtonContainer,
-  Icon: ButtonIcon,
-};
+export { Button };
```

Comment:
you can write this in one line 
```
export {Button, IButtonProps} from "./Button";
```

Reply:
You can... but: https://github.com/kadena-community/kadena.js/pull/652

Reply:
Ah I was mixing up things. Changed it now! And I changed it for all components in the PR I mentioned :D